backend/spotihue/hue.py

```diff
@@ -9,103 +9,149 @@
 
 class HueLight(phue.Light):
     """
-    A hue light that knows whether it is a white light.
+    A class that extends phue.Light. It represents a Hue light that
+    provides information about whether it is a white (non-color) light
+    and offers access to its corresponding RGB color values.
     """
 
     def __init__(self, bridge, light_id):
         super().__init__(bridge, light_id)
         self._white_light = None
+        self._rgb = None
 
     @property
     def white_light(self) -> bool:
-        """
+        """Property method to determine whether the Hue light is a white (non-color) light.
+
         Returns:
-            bool: Whether hue light is a white (non-color) light.
+            bool: True if the light is a white light, False if it's a color light.
         """
         if self._white_light is None:
             try:
-                _ = self._get('colormode')
+                _ = self._get("colormode")
                 self._white_light = False
             except KeyError:
                 self._white_light = True
 
         return self._white_light
 
+    @property
+    def rgb(self) -> Tuple[int, int, int]:
+        """Property method to return the Hue light's RGB values.
+        Converts normalized xy values to RGB color values. The x and y
+        values are assumed to be within the range [0, 1], where x + y <= 1.
+
+        Returns:
+            Tuple[int, int, int]: The RGB value as a tuple of integers (R, G, B).
+        """
+        if self._rgb is None:
+            try:
+                x, y = self.xy
+                r = int(x * 255)
+                g = int(y * 255)
+                b = int((1 - x - y) * 255)
+                self._rgb = (r, g, b)
+            except (AttributeError, ValueError):
+                self._rgb = None
```

Comment:
These x and y values are coming straight from the user's bridge, so I think we can assume they're valid. I think it'd be better to not do a try/except here that silently sets `self._rgb` to `None`, and just guarantee that the value returned here is not None.

Reply:
My only concern is if self.xy returns `None`. From the phue.py implementation line 228 https://github.com/studioimaginaire/phue/blob/master/phue.py, I'm wondering if that could happen? Then, wouldn't we want the try?

Reply:
We discussed leaving this as is to handle an error thrown for the white lights xy.